packages/ember-metal/tests/set_properties_test.js

```diff
@@ -1,4 +1,5 @@
-import { setProperties } from '..';
+import { CoreObject } from 'ember-runtime';
```

Comment:
In general we shouldn't be importing from `ember-runtime` from within `ember-metal`.

The tests in ember-metal will need to be updated to use POJO's instead of `CoreObject`...

Reply:
Issue does not happen with POJO objects it only happens with CoreObject, so moving back to CoreObject test